### PR TITLE
Migrate existing documentation for API, CLI and UMGAP

### PR DESCRIPTION
**Pull request by tibvdm**
This PR will add the documentation webpages

### CODE REVIEW

File: src/components/pages/apidocs/Pept2ProtPage.vue [hunk 1]
```diff
@@ -0,0 +1,311 @@
+<template>
+    <v-container style="font-size: 90%;">
+        <h1 class="font-weight-light">
+            <Initialism>POST</Initialism> /api/v1/pept2prot
+        </h1>
+        <h3 class="font-weight-regular grey--text">Returns the set of UniProt entries containing a given tryptic peptide.</h3>
```
Comment (pverscha) [inline src/components/pages/apidocs/Pept2ProtPage.vue]: Try to increase the contrast of the text on this line, it's a little hard to read against the background.

image

**Review by pverscha — DISMISSED** [review]
Not a lot of remarks, the only thing that we should improve is the contrast between the text and the background. The main color of the text is now mostly a dark gray, but I think it would be better if all large chunks of text are black.

image

**Comment by tibvdm** [comment]
Links are added and v-card contrast should now be fixed everywhere.

**Review by pverscha — COMMENTED** [review]
Contrast is much better now! :)

**Comment by pverscha** [comment]
During my work on adding the documentation pages for the desktop app, I stumbled upon a few minor things that I wanted to write down:

* [x] Multiple components have the same name. They are all kept in a separate directory, so this isn't really a technical issue, but I think this could cause confusion. My suggestion is to rename the components called `OverviewPage` to, for example, `MetagenomicsOverviewPage`.
* [x] In the `assets` directory, all images (and other assets) required for the documentation pages are stored in one big directory. This is fine as long as there are only a few assets, but at this point it starts to become a bit "cluttered". It's better to structure the assets directory a bit more and move the images to subdirectories per documentation category (api, cli, ...).
* [x] The "umgap translate" page of the UMGAP documentation does not correctly capitalize the "usage" title.
image

**Pull request merged**